R/bundlePython.R

```diff
@@ -0,0 +1,134 @@
+# Create anonymous function that we can later call to get all needed python
+# metdata for the manifest
+pythonConfigurator <- function(python,
+                               forceGenerate = FALSE,
+                               condaMode = FALSE) {
```

Comment:
Given that `condaMode` is always `FALSE`, is it ok to remove?

Reply:
Yes. 🔥